tests/unittests/test_model_zoo.py

```diff
@@ -49,6 +49,27 @@ def test_ssd_models():
             net.initialize()
         net(x)
 
+def test_resnet_v1b():
```

Comment:
seems better merge with classification case.

Reply:
sure. The input size is 224x224. current classification test only seems only having cifar models

Reply:
Feel free to rename it when imagenet model test case is covered :)